apps/omg/test/omg/state/persistence_test.exs

```diff
@@ -226,7 +241,8 @@ defmodule OMG.State.PersistenceTest do
         owner: owner.addr,
         currency: currency,
         amount: amount,
-        blknum: blknum
+        blknum: blknum,
+        eth_height: 1
```

Comment:
why do we need this?

Reply:
it's required for `EthEvent.insert_deposit!/1`:
```elixir
    %{
      root_chain_txhash: root_chain_txhash,
      log_index: log_index,
      blknum: blknum,
      owner: owner,
      eth_height: eth_height,
      currency: currency,
      amount: amount
    } = params
```